modules/backend/widgets/calendar/assets/less/calendar.less

```diff
@@ -1,16 +1,30 @@
+
+
+// Importing file while using 
+// @import "../../../../../system/assets/ui/less/button.variables.less";
```

Comment:
Commented out?

Reply:
Yes, my intention was to get the brand colors of Winter, but I think the assets needs to be compile instead of directly use the less file.

Reply:
The calendar css is now compiled with `winter:util compile less` command.  
The winter's button colors are used to style calendar widget buttons.